packages/main/src/themes/MultiComboBoxPopover.css

```diff
@@ -31,4 +31,8 @@
 
 .ui5-mcb-select-all-checkbox::part(label) {
 	font-family: var(--sapFontBoldFamily);
-}
\ No newline at end of file
+}
+
+:host([readonly]) ::slotted([ui5-mcb-item]) {
```

Comment:
should not be a valid case

Reply:
mobile

Reply:
<img width="440" alt="image" src="https://github.com/user-attachments/assets/4adc9ab4-2798-46d1-b1ca-af09258ad664">